controllers/ajax.py

```diff
@@ -707,11 +707,12 @@ def getSphinxBuildStatus():
         if row['status'] in ['QUEUED', 'ASSIGNED','RUNNING', 'COMPLETED']:
             results['status'] = row['status']
         else:  # task failed
-            results['status'] = 'failed'
+            results['status'] = row['status']
             tb = db(db.scheduler_run.task_id == row.id).select().first()['traceback']
             results['traceback']=tb
     else:
         results['status'] = 'failed'
+        results['info'] = 'no row'
     return json.dumps(results)
```

Comment:
This looks like it is unrelated to the purpose of the PR -- but seems like valid cleanup??